backend/engine/processor/processor.py

```diff
@@ -156,7 +156,7 @@ def process_plugins(self, images: dict, services: dict) -> None:
                         results.details = []
 
                         # Use the CycloneDX format by default when not using veracode_sbom tool
-                    elif results.type == PluginType.SBOM.value and plugin != "veracode_sbom":
+                    elif results.type == PluginType.SBOM.value and not results.success and plugin != "veracode_sbom":
```

Comment:
don't try processing the sbom if there are no results (usually occurs if there was some sort of scanning error from trivy-sbom)